spec/integration/appointments_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'swagger_helper'
+
+RSpec.describe 'appointments', type: :request do
+  path '/appointments/create' do
+    post('create appointments') do
+      tags 'api/appointments'
+      response(200, 'successful') do
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+        run_test!
+      end
+    end
+  end
+
+  path '/{user_id}/appointments/' do
```

Comment:
You have added the requested url : https://localhost:3000/1/appointments/
error raised becasue of wrong path , your actual routes in config file is like 
https://localhost:3000/users/1/appointments/.
pelase change the path of the documentation to avoid error.
![image](https://github.com/MudasirSherwani/Book-an-Appointment/assets/86018834/469143cb-6d7b-40e2-b830-51fc271d9361)